src/components/UserProfile/elements/EmptyContent.jsx

```diff
@@ -1,41 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
+import { Button } from 'reactstrap';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faPlus } from '@fortawesome/free-solid-svg-icons';
 
 function EmptyContent({ children, onClick, showPlusIcon }) {
   const onKeyDown = (e) => { if (e.key === 'Enter') onClick(); };
 
-  const commonProps = {
-    className: 'd-flex align-items-center p-3 bg-light rounded text-muted w-100',
-    style: {
-      cursor: onClick ? 'pointer' : null,
-    },
-  };
-
   const interactiveProps = {
     onClick,
     onKeyDown,
     role: 'button',
     tabIndex: 0,
   };
 
-  let props;
-
-  if (onClick) {
-    props = {
-      ...commonProps,
-      ...interactiveProps,
-    };
-  } else {
-    props = commonProps;
-  }
-
   return (
-    <div {...props}>
-      {showPlusIcon ? <FontAwesomeIcon className="ml-1 mr-3" icon={faPlus} /> : null}
-      {children}
+    <div>
+      {onClick ? (
+        <Button
+          color="link"
```

Comment:
Buttons are supposed to have a `type` attribute, right?  Think the a11y linter complains about that...

Reply:
Maybe reactstrap makes it `type="button"` by default, I suppose.

Reply:
The source for Button https://github.com/reactstrap/reactstrap/blob/master/src/Button.js

Surprisingly the type attribute doesn't seem to be addressed at all there, though if specified it will pass through. I'll swap this with the Paragon Button

Reply:
If reactstrap is our first choice to avoid maintenance, any use in trying to open a PR against reactstrap to try to address issues like this? (This proposal would be in parallel, in addition to switching to Paragon for solving today's issue.)